src/framework/entity.js

```diff
@@ -20,29 +40,149 @@ import { Application } from './application.js';
  * Entity and are attached (e.g. `this.entity.model`) ComponentSystems allow access to all Entities
  * and Components and are attached to the {@link Application}.
  *
- * @property {AnimComponent} [anim] Gets the {@link AnimComponent} attached to this entity. [read only]
- * @property {AnimationComponent} [animation] Gets the {@link AnimationComponent} attached to this entity. [read only]
- * @property {AudioListenerComponent} [audiolistener] Gets the {@link AudioListenerComponent} attached to this entity. [read only]
- * @property {ButtonComponent} [button] Gets the {@link ButtonComponent} attached to this entity. [read only]
- * @property {CameraComponent} [camera] Gets the {@link CameraComponent} attached to this entity. [read only]
- * @property {CollisionComponent} [collision] Gets the {@link CollisionComponent} attached to this entity. [read only]
- * @property {ElementComponent} [element] Gets the {@link ElementComponent} attached to this entity. [read only]
- * @property {LayoutChildComponent} [layoutchild] Gets the {@link LayoutChildComponent} attached to this entity. [read only]
- * @property {LayoutGroupComponent} [layoutgroup] Gets the {@link LayoutGroupComponent} attached to this entity. [read only]
- * @property {LightComponent} [light] Gets the {@link LightComponent} attached to this entity. [read only]
- * @property {ModelComponent} [model] Gets the {@link ModelComponent} attached to this entity. [read only]
- * @property {ParticleSystemComponent} [particlesystem] Gets the {@link ParticleSystemComponent} attached to this entity. [read only]
- * @property {RenderComponent} [render] Gets the {@link RenderComponent} attached to this entity. [read only]
- * @property {RigidBodyComponent} [rigidbody] Gets the {@link RigidBodyComponent} attached to this entity. [read only]
- * @property {ScreenComponent} [screen] Gets the {@link ScreenComponent} attached to this entity. [read only]
- * @property {ScriptComponent} [script] Gets the {@link ScriptComponent} attached to this entity. [read only]
- * @property {ScrollbarComponent} [scrollbar] Gets the {@link ScrollbarComponent} attached to this entity. [read only]
- * @property {ScrollViewComponent} [scrollview] Gets the {@link ScrollViewComponent} attached to this entity. [read only]
- * @property {SoundComponent} [sound] Gets the {@link SoundComponent} attached to this entity. [read only]
- * @property {SpriteComponent} [sprite] Gets the {@link SpriteComponent} attached to this entity. [read only]
  * @augments GraphNode
```

Comment:
do we still need augments tag?

Reply:
Last time I checked, yeah, we do - either for JSDoc (for showing all the inherited stuff) or for the TS declarations. I can't remember which. But I will keep investigating if we can dump that somehow. Just not in this PR. 😄